lib/Server.js

```diff
@@ -405,7 +405,7 @@ class Server {
       throw new Error('Watching remote files is not supported.');
     } else if (Array.isArray(contentBase)) {
       contentBase.forEach((item) => {
-        if (isAbsoluteUrl(String(item))) {
+        if (isAbsoluteUrl(String(item)) || typeof item === 'number') {
```

Comment:
Why we add `typeof item === 'number'`?

Reply:
@evilebottnawi it's already blocking number values in options.js accepting only strings, in case in future refactors if we change the type of array items to accept any type we still need it to throw an exception if the array item is number type.

Reply:
@evilebottnawi we can remove number type from options.json 
```
  "contentBase": {
      "anyOf": [
        {
          "enum": [false]
        },
        {
          "type": "number"
        },
        {
          "type": "string"
        },
        {
          "type": "array",
          "items": {
            "type": "string"
          },
          "minItems": 1
        }
      ]
    },
```
and remove ` || typeof item === 'number'` checks and update tests to check for the exception ```"should be {String|Array} (https://webpack.js.org/configuration/dev-server/#devservercontentbase)"```
what do you think ?

Reply:
Let's do it next version